cmake/test_util.cmake

```diff
@@ -23,10 +24,13 @@ function(create_standard_test)
   set(singleValues NAME WORKING_DIRECTORY)
   set(multiValues SOURCES)
   include(CMakeParseArguments)
-  cmake_parse_arguments(${prefix} " " "${singleValues}" "${multiValues}" ${ARGN})
+  cmake_parse_arguments(${prefix} " " "${singleValues}" "${multiValues}"
+                        ${ARGN})
```

Comment:
Is there a reason the cmake is getting formatted at a hard limit of 80 characters?  I'm fully supportive of being consistent across our cmake but if this isn't needed as a part of the tridiagonal solver, could we save these kinds of edits for another PR?

Reply:
I found the problem. It was the cmake linter in my editor configuration. I have reverted the formatting changes and pushed the changes.